slides/08_Data_Fetching.html

```diff
@@ -501,6 +383,8 @@ <h2>Recap</h2>
             <li>Why fetching data is tricky with React render mechanism</li>
             <li>How to fetch data in React with a library</li>
             <li>Display details relating to loading & errors</li>
+            <li>Using useMutation for data manipulation</li>
+            <li>How to update the local cache without refetch</li>
```

Comment:
This is the "optimistic UI" concept, I think we should add that name somewhere (here or before where it is shown). It's often to be found in docs under this name.